app/controllers/copy_course_from_production_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#= Controller for copying course from production to test
+class CopyCourseFromProductionController < ApplicationController
+  respond_to :html
+  before_action :require_admin_permissions
+
+  def index
+    @copied_course = nil
+  end
+
+  def copy
+    copied_course = CopyCourseFromProduction.new(params[:url])
+    pp "http://localhost:3000/courses/#{copied_course.course_slug}"
```

Comment:
We shouldn't have any `pp` output in the production rails code.

Reply:
Removed here [ecd6486](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/4914/commits/ecd6486288eafd4dff4518664b57ded7d7360153)